addon/webextension/background.js

```diff
@@ -77,10 +77,20 @@ class PersistentPageModificationEffect {
         display: inline-block;
       }
 
+      .donotdelete-revert {
+        transform: ScaleY(1);
```

Comment:
nit: `scaleY(1)` (lowercase)

Not sure if there'd be a way to extract this CSS into an external file so we could validate it using [**stylelint**](http://npm.im/stylelint) or something, but now I'm getting super nit-pickety...
Currently I just copy the contents into an external file and lint that manually, but it's not foolproof, and I'm a pretty big fool.

Reply:
No time :)